src/handlers/oracles.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
    * responses: 200, 400, 401, 403, 404, 405, 406, 501, 503
    */
   get: async (request, h) => {
-    const response = await oracle.getOracle(request.query)
+    const response = await oracle.getOracle(request.query, request.headers, request.span)
```

Comment:
Normally this is where we do the initial span.audit with action EventSdk.AuditEventAction.start --- example --> https://github.com/mojaloop/ml-api-adapter/blob/master/src/api/transfers/handler.js

Reply:
Initial span.audit with action EventSdk.AuditEventAction.start is at src/domain/oracle/oracle.js getOracle function. Please let me know if needs to come up at the src/handlers/oracles.js level.

Reply:
Yes, please. It should be consistent and handled in the handlers directly.

Reply:
Code has been refactored to have all newly added event sdk code at handlers level.